### ISSUE TITLE
Grayscale synthetic noise

### ISSUE THREAD

**Issue by pchizhov** [demonsjin/Blind2Unblind#17]
There seems to be no code for training/testing on grayscale data with synthetic Gaussian noise (BSD400, BSD68, Set12, and Urban100), the results for which are presented in Table 2. The weights for these models also are not present.

Also, it is not clear from the paper, how exactly you use Urban100 data, because it is originally sRGB and in pairs of high- and low-resolution images.

It would be great if you could comment on that.

**Comment by zejinwang** [comment]
Please refer to [Recorrupted-to-Recorrupted](https://github.com/PangTongyao/Recorrupted-to-Recorrupted-Unsupervised-Deep-Learning-for-Image-Denoising) for more details. Actually, Blind2Unblind can perform much better on tiny grayscale images. The initial learning rate should be set to 1e-3 on small train sets.
